### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by burnout87**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
-# renku-oda-annotation
\ No newline at end of file
+# renku-oda-annotation
+
+This renku plugin provides the following functionalities:
+
+* Generate astroquery specific annotations by intercepting the call to the following functions:
```
Comment (volodymyrss) [inline README.md]:
```suggestion
* Generate annotations describing access to astronomical archives and services by intercepting any call to the following methods of `astroquery` classes :
```

File: README.md [hunk 2]
```diff
@@ -1 +1,13 @@
...
+  * query_object
+  * get_images
+  * query_region
+* CLI `inspect` to analyze input notebooks and extract a set of metadata to be visualized over the knowledge graph.
```
Comment (volodymyrss) [inline README.md]:
```suggestion
* CLI `inspect` to analyze data analysis notebooks and extract a set of metadata and record it in the local knowledge graph.
```

File: renkuaqsannotation/plugin.py [hunk 3]
```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 - Viktor Gal
```
Comment (volodymyrss) [inline renkuaqsannotation/plugin.py]: we need to modify this license

Comment (burnout87) [reply renkuaqsannotation/plugin.py]: Which one shall I use instead?

Comment (volodymyrss) [reply renkuaqsannotation/plugin.py]: put yourself, remove the lines about " A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and.."

Comment (burnout87) [reply renkuaqsannotation/plugin.py]: no mention of UniGe and/or EPFL?

Comment (volodymyrss) [reply renkuaqsannotation/plugin.py]: put both in brackets after your name. It's not necessary for the license, and the requirements from the legal department do not specify this. we can adapt later common license.

**Review by volodymyrss — APPROVED** [review]

**Review by volodymyrss — APPROVED** [review]

**Pull request merged**